### PR TITLE
Pull Request for References branch

### PR DESCRIPTION
**Pull request by tcpiplab**
Adding this in as part of the final exercise in Udacity's How to Use Git And GitHub course.

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2721)

### CODE REVIEW

**Comment by tcpiplab** [comment]
Please review this, @casey-collab

File: english/references/references.md [hunk 1]
```diff
@@ -0,0 +1 @@
+You calmly explain to the Chief Marshmallow that you come from the [Land of Ice and Snow](../../norwegian/).
```
Comment (casey-collab) [inline english/references/references.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by tcpiplab** [comment]
Thanks @casey-collab, please review it again. I fixed the link.

**Comment by casey-collab** [comment]
Looks great! :ice_cream:

**Pull request merged**